src/plugins/intel_cpu/src/nodes/kernels/dft_uni_kernel.cpp

```diff
@@ -0,0 +1,292 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "dft_uni_kernel.hpp"
+
+
+using namespace dnnl::impl;
+using namespace dnnl::impl::utils;
+using namespace dnnl::impl::cpu::x64;
+
+#define GET_OFF_DFT(field) offsetof(jit_args_dft, field)
+#define GET_OFF_FFT(field) offsetof(jit_args_fft, field)
+
+namespace ov {
+namespace intel_cpu {
+
+template <cpu::x64::cpu_isa_t isa>
+jit_uni_dft_kernel_f32<isa>::jit_uni_dft_kernel_f32(jit_dft_config_params jcp) : jcp_(jcp), jit_uni_dft_kernel(), jit_generator() {}
+
+template <cpu::x64::cpu_isa_t isa>
+void jit_uni_dft_kernel_f32<isa>::create_ker() {
+    jit_generator::create_kernel();
+    ker_ = (decltype(ker_))jit_ker();
+}
+
+template <cpu::x64::cpu_isa_t isa>
+void jit_uni_dft_kernel_f32<isa>::generate() {
+    this->preamble();
+
+    mov(reg_src, ptr[reg_params + GET_OFF_DFT(src)]);
+    mov(reg_dst, ptr[reg_params + GET_OFF_DFT(dst)]);
+    mov(reg_twiddles, ptr[reg_params + GET_OFF_DFT(twiddles)]);
+    mov(reg_work_amount, ptr[reg_params + GET_OFF_DFT(work_amount)]);
+    mov(reg_index, ptr[reg_params + GET_OFF_DFT(index)]);
+
+    Xbyak::Label main_loop_label;
+    Xbyak::Label main_loop_end_label;
+    Xbyak::Label tail_loop_label;
+    Xbyak::Label tail_loop_end_label;
+
+    mov(aux_reg_work_amount, reg_work_amount);
+    uni_vpxor(vmm_sum, vmm_sum, vmm_sum);
+
+    int step = vlen / 8;
+
+    L(main_loop_label);
+    {
+        cmp(aux_reg_work_amount, step);
+        jl(main_loop_end_label, T_NEAR);
+
+        uni_vpshufd(vmm_data, ptr[reg_src], 0b01000001);
```

Comment:
uni_vpshufd uses just 8 bit in the order operand. 0b01000001 - why do you need other here?